internal/rbac/access.go

```diff
@@ -0,0 +1,131 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rbac
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	v1 "k8s.io/api/authorization/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	serviceAccountFmtStr = "system:serviceaccount:%s:%s"
+)
+
+type Reviewer struct {
+	client kubernetes.Interface
+}
+
+func NewReviewer(c kubernetes.Interface) *Reviewer {
+	return &Reviewer{
+		client: c,
+	}
+}
+
+// AllSubjectAccessReviewsAllowed checks if all of subjectAccessReviews are explicitly allowed.
```

Comment:
The comment should specify that the function returns denied SARs.

Reply:
true! I'll add that in :)